common/device-model.cpp

```diff
@@ -1221,7 +1225,11 @@ namespace rs2
         auto record_button_color = is_recording ? light_blue : light_grey;
         ImGui::PushStyleColor(ImGuiCol_Text, record_button_color);
         ImGui::PushStyleColor(ImGuiCol_TextSelectedBg, record_button_color);
-        if (ImGui::ButtonEx(record_button_name.c_str(), device_panel_icons_size, (disable_record_button_logic(is_streaming, is_playback_device)) ? ImGuiButtonFlags_Disabled : 0))
+        bool button_disabled = disable_record_button_logic(is_streaming, is_playback_device);
+        if (button_disabled)
+            ImGui::PushStyleColor(ImGuiCol_Text,ImVec4(0.5f,0.5f,0.5f,1.f));
```

Comment:
Why do we need this?

Reply:
the flag ImGuiButtonFlags_Disabled  was removed so I used ImGuiItemFlags_Disabled flag instead and it wasn't changing the button color (to dark gray). I searched more and found ImGui::BeginDisabled() function that does it better and used it instead.